### PR TITLE
Update installation instructions for conda

### PR DESCRIPTION
**Pull request by gavinmischler**
#### What does this implement/fix? Explain your changes.
Given the discussion on #314, we will be changing the installation of dependencies on the conda-forge version, so this changes the instructions.

### CODE REVIEW

**Comment by agramfort** [comment]
wait a minute multiviewica already in conda-forge

```
Looking for: ['multiviewica']

pkgs/r/osx-64            [====================] (00m:00s) No change
pkgs/r/noarch            [====================] (00m:00s) No change
pkgs/main/noarch         [====================] (00m:00s) Done
pkgs/main/osx-64         [====================] (00m:00s) Done
conda-forge/noarch       [====================] (00m:01s) Done
conda-forge/osx-64       [====================] (00m:04s) Done
Transaction

  Prefix: /Users/alex/miniconda3

  Updating specs:

   - multiviewica


  Package            Version  Build           Channel                  Size
─────────────────────────────────────────────────────────────────────────────
  Install:
─────────────────────────────────────────────────────────────────────────────

  fastsrm              0.0.4  pyhd8ed1ab_0    conda-forge/noarch      20 KB
  multiviewica         0.0.1  pyhd8ed1ab_0    conda-forge/noarch      11 KB

  Change:
─────────────────────────────────────────────────────────────────────────────

  certifi          2021.10.8  py38h50d1736_1  installed
  certifi          2021.10.8  py38h50d1736_2  conda-forge/osx-64     145 KB

  Upgrade:
─────────────────────────────────────────────────────────────────────────────

  ca-certificates  2022.3.18  hecd8cb5_0      installed
  ca-certificates  2022.3.29  hecd8cb5_0      pkgs/main/osx-64       118 KB

  Summary:

  Install: 2 packages
  Change: 1 packages
  Upgrade: 1 packages

  Total download: 293 KB

─────────────────────────────────────────────────────────────────────────────

Confirm changes: [Y/n]
```

**Comment by gavinmischler** [comment]
Oh awesome, I wasn't aware it had been added. I updated the installation guide

**Comment by hugorichard** [comment]
Indeed ! I believe I was the one putting it in conda-forge. I think it is up to date !

File: docs/install.rst
Comment (hoechenberger) [inline docs/install.rst:59]: I don't think even mentioning the term "feedstock" is necessary here. All that users care about is the **package**. The feedstock is really just the repository containing all the build info, which isn't of much use to a user. I would suggest to simply supply the conda install command, and provide a link to conda-forge.org for those who don't know about this yet. 
Just my 2 ct :)

**Comment by hoechenberger** [comment]
> Indeed ! I believe I was the one putting it in conda-forge. I think it is up to date ! Sorry for the long reaction time...

I can help you co-maintain the feedstock if you want to. I'll ask the bot to open a respective PR and then you're free to merge or ignore ;)

**Comment by hugorichard** [comment]
> > Indeed ! I believe I was the one putting it in conda-forge. I think it is up to date ! Sorry for the long reaction time...
> 
> I can help you co-maintain the feedstock if you want to. I'll ask the bot to open a respective PR and then you're free to merge or ignore ;)

Thanks ! I'll be happy to co-maintain the feedstock with you. The PR says some checks were not successful. Should I merge anyway ? 
Is there a way to declare you as a co-maintainer so that you can change things without my approval ?

**Comment by hoechenberger** [comment]
> Thanks ! I'll be happy to co-maintain the feedstock with you. The PR says some checks were not successful. Should I merge anyway ? Is there a way to declare you as a co-maintainer so that you can change things without my approval ?

Just merge https://github.com/conda-forge/multiviewica-feedstock/pull/2
It's only red because it doesn't run CI for changes like these :)
Once merged, I'll be able to make modifications to the feedstock without your approval, yes

File: docs/install.rst
Comment (gavinmischler) [reply docs/install.rst:59]: Yeah I agree. Our installation guide is already complex enough so I agree that this is just unnecessary

**Review by agramfort — APPROVED** [review]

**Pull request merged**
